lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_validate_ci_build.rb

```diff
@@ -2,7 +2,7 @@ module Fastlane
   module Actions
     class IosValidateCiBuildAction < Action
       def self.run(params)
-        require_relative '../../helper/ios/ios_git_helper'
+        return_value '../../helper/git_helper'
```

Comment:
```suggestion
        require_relative '../../helper/git_helper'
```

Reply:
Wow... not sure how this ended up like that. Thanks @spencertransier !

Reply:
And, even better, thanks for doing the work to check this was unnecessary and deleting it via https://github.com/wordpress-mobile/release-toolkit/pull/586 !